### ISSUE TITLE
Perform internal a11y pass over design system documentation site

### ISSUE THREAD

**Issue by pattyde** [govau/designsystem#54]
Prior to the official launch on April 23rd, we should do a final a11y pass over the design system documentation site. 

We've allocated 1 day for testing + additional time as needed to fix any high priority a11y issues that are uncovered.

**WAVE test**
https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh

**Keyboard test**
Use keyboard navigation for all elements on page. No keyboard traps.

**Screenreader test**
Use apple screen reader to read pages. mostly looking for weird text or missing explanations.

- [x] WAVE test - Home
- [x] WAVE test - Content page
- [x] WAVE test - Components page
- [x] WAVE test - Component / button
- [x] WAVE test - Download page
- [x] Keyboard test - Home
- [x] Keyboard test - Content page
- [x] Keyboard test - Components page
- [x] Keyboard test - Component / button
- [x] Keyboard test - Download page
- [x] Screenreader test - Home
- [x] Screenreader test - Content page
- [x] Screenreader test - Components page
- [x] Screenreader test - Component / button
- [x] Screenreader test - Download page
